src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import duke.command.Command;
+
+/**
+ * Represents chatbot duke
+ */
+public class Duke {
+
+    private static final String filePath = "./data/duke.txt";
+    private TaskList tasks;
+    private Storage storage;
+    private Ui ui;
+
+    /**
+     * Initialize duke
```

Comment:
The [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#comments) specifies Javadoc comments need proper punctuation. Perhaps it would be better to reformat all your Javadoc comments to follow this.